KSP-Backend/KSP/mvp/models.py

```diff
@@ -0,0 +1,37 @@
+import datetime
+from django.db import models
+
+YEAR_CHOICES = []
+for year in range(2016, (datetime.datetime.now().year + 1)):
+    YEAR_CHOICES.append((year, year))
+
+
+class Notes(models.Model):
+    course_id = models.CharField(max_length=10)
+    batch = models.IntegerField(choices=YEAR_CHOICES)
+    course_instructor = models.CharField(max_length=100)
+    uploaded_at = models.DateTimeField(auto_now=True)
+    gdrive_link = models.URLField(max_length=500)
+
+class Feedback(models.Model):
+    name = models.CharField(max_length=100)
+    course_id = models.CharField(max_length=10)
+    course_instructor = models.CharField(max_length=100)
+    batch = models.IntegerField(choices=YEAR_CHOICES)
+    created_date = models.DateTimeField(auto_now_add=True)
+    updated_date = models.DateTimeField(auto_now=True)
+    feedback = models.TextField()
+
+class PreviousYearQuestions(models.Model):

```

Comment:
This model is exactly similar to the `Notes` model. 

The `notes` model should be enough all kinds of notes(even Previous Year Questions since we just use the drive links for notes).